Octokit/Models/Response/Deployment.cs

```diff
@@ -14,48 +14,48 @@ public class Deployment
         /// <summary>
         /// Id of this deployment.
         /// </summary>
-        public int Id { get; set; }
+        public int Id { get; protected set; }
 
         /// <summary>
         /// 
         /// </summary>
-        public string Sha { get; set; }
+        public string Sha { get; protected set; }
 
         /// <summary>
         /// The API URL for this deployment.
         /// </summary>
-        public string Url { get; set; }
+        public string Url { get; protected set; }
 
         /// <summary>
         /// The <seealso cref="User"/> that created the deployment.
         /// </summary>
-        public User Creator { get; set; }
+        public User Creator { get; protected set; }
 
         /// <summary>
         /// JSON payload with extra information about the deployment.
         /// </summary>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
```

Comment:
We shouldn't initialize them in the ctors for these types because they'll just get replaced when SimpleJson deserializes the json into an object. It uses the private setters to set the collection.